src/twisted/python/__init__.py

```diff
@@ -17,6 +17,13 @@
     "Please use constantly from PyPI instead.",
     "twisted.python", "constants")
 
+
+deprecatedModuleAttribute(
+    Version("Twisted", "NEXT", 0, 0),
```

Comment:
I was talking about our current documentation in docs/core/development/policy/compatibility-policy.rst